aldourtusuastegui/MovieTime
# Android |Feature movies | Add offline data persistence

### **What is solving?** 
add offline data persistence when network is not available.

### **Solution description**
-created Room entities.
-created Dao.
-created Database.
-useCase is validating if device has network.
-added funs to get and insert movies from db.
-cleaned code.

### **Steps to verify changes**
-Run the application or see attached video


https://github.com/aldourtusuastegui/MovieTime/assets/13209412/cd09957a-7f76-4f13-b30b-4dbdf6eae7dc
